one_fm/setup.py

```diff
@@ -20,6 +22,7 @@
 
 def after_install():
 	create_custom_fields(get_custom_fields())
+	delete_custom_fields_after_install()
```

Comment:
We dont need to delete a field which is not even present in the field list, You can remove this from here